less/gmcq.less

```diff
@@ -1,5 +1,46 @@
 .gmcq-component {
+    &.gmcq-column-layout {
+        .gmcq-widget {
+            // Font size set to 0 so there's no gap between items
+            font-size: 0;
+            text-align: center;
+        }
+
+        .gmcq-item {
+            display: inline-block;
+            float: none;
+            vertical-align: top;
+
+            .dir-rtl & {
+                float: none;
+            }
+
+            &.even,
+            &.odd {
+                label {
+                    margin: 2.5%;
+
+                    .dir-rtl & {
+                        margin: 2.5%;
+                    }
 
+                    @media (max-width: @device-width-medium) {
+                        margin: 0;
+                    }
```

Comment:
I believe this media query is redundant because the `gmcq-column-layout` class is only used on large devices?